### PR TITLE
Updates

### PR DESCRIPTION
**Pull request by differentialderek**
## Overview
Updates on brain dump. I'm thinking in a standard form to have a syntax section, and inputs section, and maybe a notes section. Something like that?

### Which JIRA issue does this PR relate to? If there is not a JIRA issue addressing this work, please create one now and add the link here.
https://rchain.atlassian.net/browse/RHOL-960

### Complete this checklist before you submit the PR
- [X] This PR contains no more than 200 lines of code, excluding test code.
- [X] This PR meets [RChain development coding standards](https://rchain.atlassian.net/wiki/spaces/DOC/pages/28082177/Coding+Standards).
- [X] If this PR adds a new feature, this PR includes tests related to this feature.
- [X] You assigned one person to review this PR

### If you are not a member of the core development team, please confirm:
- [ ] You signed the commit. Merging requires a signature. Please see the [developer wiki](https://rchain.atlassian.net/wiki/spaces/DOC/pages/498630673/How+to+sign+commits+to+rchain+rchain) for instructions.
- [ ] Your GitHub account is also an account with [Travis CI](https://travis-ci.org). Unit tests will not run on your PR unless you have an account with Travis. Merge requires passed unit tests.

### Notes
Optional. Add any notes on caveats, approaches you tried that didn't work, or anything else.

### CODE REVIEW

**Comment by differentialderek** [comment]
"Standard form" may not be as nice and concise as I thought, since there aren't a lot of standard things. Standard form will make sense on methods though.

**Comment by JoshOrndorff** [comment]
Yeah I agree that the standard form will be useful primarily in for methods.

File: rholang/reference_doc/bundles/README.md [hunk 1]
```diff
@@ -1,7 +1,22 @@
...
+- `chan` is read only
+
+### No read, no write
+- `bundle0{ chan }`
+- `chan` can't be read or written on
+- Useful in some cases
+  - EG here
+
+### bundle: bundle read write
+- `bundle-{ chan }`
```
Comment (JoshOrndorff) [inline rholang/reference_doc/bundles/README.md]: @differentialderek I think this is the line that @goral09 Was commenting on. The `-` here is accidental, right?

Comment (differentialderek) [reply rholang/reference_doc/bundles/README.md]: Ah, yes, thank you.

**Comment by differentialderek** [comment]
> This is an error. `bundle-{ chan }` is read only channel.

Thank you for catching that.

**Pull request merged**